.github/workflows/github-actions.yml

```diff
@@ -33,7 +33,8 @@ jobs:
         run: |
           python3 -m unittest discover -v ./test/
           python3 -m unittest discover -v ./test/assertion/*
-          python3 -m unittest discover -v ./test/simulator/*
+          python3 -m unittest discover -v ./test/simulator/pauli_circuit/
+          python3 -m unittest discover -v ./test/simulator/circuit_drawer/
```

Comment:
Somehow the wildcard did not work...

Reply:
@mitz1012 
I found the solution to this problem. Please review https://github.com/QuantestPy/quantestpy/pull/168 first. Thanks!